docs/config-file-curations-yml.md

```diff
@@ -25,7 +25,7 @@ Curations can be used to:
   case of a fork of an upstream Open Source project, or a copy of the code in this project's repository. 
 * set the _declared_license_mapping_ property:
   * Packages may have declared license string values which cannot be parsed to SpdxExpressions. In some cases this can
-    be fixed by mapping these strings to a valid license. If multiple curations declare license mapping they get
+    be fixed by mapping these strings to a valid license. If multiple curations declare license mapping, they get
```

Comment:
Should also say "mappings" (plural), or maybe "a license mapping". The commit message needs to be adjusted then to not only talk about punctuation.